modules/dxp/apps/portal-workflow-kaleo-forms/portal-workflow-kaleo-forms-web/src/main/java/com/liferay/portal/workflow/kaleo/forms/web/internal/asset/KaleoProcessAssetRendererFactory.java

```diff
@@ -113,11 +111,6 @@ public boolean hasPermission(
 	@Reference
 	private KaleoProcessLocalService _kaleoProcessLocalService;
 
-	@Reference(
-		target = "(javax.portlet.name=" + KaleoFormsPortletKeys.KALEO_FORMS_ADMIN + ")"
-	)
-	private Portlet _portlet;
```

Comment:
Do not remove this one, please see the reason why [here](https://github.com/liferay-workflow/liferay-portal/pull/366/files)

cc: @pedro-oliveira446 @diogomelo09 @guilhermedcamacho @gabrielwas